public/javascripts/SVValidate/src/label/LabelControl.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Handles the hiding and showing of labels in the Google StreetView panorama.
+ * This is also called by the Keyboard class to deal with hiding the label
+ * via keyboard shortcuts.
+ * @returns {LabelControl}
+ * @constructor
+ */
+function LabelControl () {
```

Comment:
You changed the names in the html and css from "label control" to "label visibility control", but you didn't do it in the JavaScript yet.

Reply:
Fixed